ovisbot/extensions/ctf/ctf.py

```diff
@@ -124,13 +127,21 @@ async def archive(self, ctx, ctf_name):
         ctf = CTF.objects.get({"name": ctf_name})
         category = discord.utils.get(ctx.guild.categories, name=ctf_name)
         ctfrole = discord.utils.get(ctx.guild.roles, name="Team-" + ctf_name)
-
+        
         if ctfrole is not None:
             await ctfrole.delete()
+
         for c in category.channels:
+
+            challenge = next((ch for ch in ctf.challenges if ch.name == c.name), None)
+            if not not challenge and challenge.solved_at:
```

Comment:
`if not not challenge`? Why not just `if challenge is not None` 😄

Reply:
valid point :smile: my bad